File: Anole/HTMLRenderBookmark.vb
Imports System

Namespace Anole
    ''' <summary>
    ''' Summary description for HTMLRenderBookmark.
    ''' </summary>
    Public Class HTMLRenderBookmark
        Inherits HTMLRenderItem
        Public Sub New(element As HTMLElement)
            Me.Element = element
        End Sub
        Public Overloads Overrides Sub Paint(g As System.Drawing.Graphics)
            'do nothing, I'm a bookmark
        End Sub
        
    End Class
End Namespace

File: Anole/HTMLRenderBookmark.cs
using System;

namespace Anole
{
	/// <summary>
	/// Summary description for HTMLRenderBookmark.
	/// </summary>
	public class HTMLRenderBookmark : HTMLRenderItem
	{
		public HTMLRenderBookmark(HTMLElement element)
		{
			this.Element=element;
		}
		public override void Paint(System.Drawing.Graphics g)
		{
			//do nothing, I'm a bookmark
		}

	}
}
